semconv/src/main/java/io/opentelemetry/semconv/ResourceAttributes.java

```diff
@@ -971,40 +1432,80 @@ public final class ResourceAttributes {
 
-  /** The version string of the auto instrumentation agent or distribution, if used. */
+  /**
+   * The version string of the auto instrumentation agent or distribution, if used.
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.TelemetryIncubatingAttributes#TELEMETRY_DISTRO_VERSION}
+   *     attribute.
+   */
   @Deprecated
   public static final AttributeKey<String> TELEMETRY_DISTRO_VERSION =
       stringKey("telemetry.distro.version");
 
-  /** Additional description of the web engine (e.g. detailed version and edition information). */
+  /**
+   * Additional description of the web engine (e.g. detailed version and edition information).
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.WebengineIncubatingAttributes#WEBENGINE_DESCRIPTION}
+   *     attribute.
+   */
   @Deprecated
   public static final AttributeKey<String> WEBENGINE_DESCRIPTION =
       stringKey("webengine.description");
 
-  /** The name of the web engine. */
+  /**
+   * The name of the web engine.
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.WebengineIncubatingAttributes#WEBENGINE_NAME}
+   *     attribute.
+   */
   @Deprecated public static final AttributeKey<String> WEBENGINE_NAME = stringKey("webengine.name");
 
-  /** The version of the web engine. */
+  /**
+   * The version of the web engine.
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.WebengineIncubatingAttributes#WEBENGINE_VERSION}
+   *     attribute.
+   */
   @Deprecated
   public static final AttributeKey<String> WEBENGINE_VERSION = stringKey("webengine.version");
 
-  /** The name of the instrumentation scope - ({@code InstrumentationScope.Name} in OTLP). */
+  /**
+   * The name of the instrumentation scope - ({@code InstrumentationScope.Name} in OTLP).
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.OtelIncubatingAttributes#OTEL_SCOPE_NAME} attribute.
+   */
   @Deprecated
   public static final AttributeKey<String> OTEL_SCOPE_NAME = stringKey("otel.scope.name");
 
-  /** The version of the instrumentation scope - ({@code InstrumentationScope.Version} in OTLP). */
+  /**
+   * The version of the instrumentation scope - ({@code InstrumentationScope.Version} in OTLP).
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.OtelIncubatingAttributes#OTEL_SCOPE_VERSION} attribute.
+   */
   @Deprecated
   public static final AttributeKey<String> OTEL_SCOPE_VERSION = stringKey("otel.scope.version");
 
   /**
    * Deprecated, use the {@code otel.scope.name} attribute.
    *
    * @deprecated Deprecated, use the `otel.scope.name` attribute.
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.OtelIncubatingAttributes#OTEL_LIBRARY_NAME} attribute.
```

Comment:
duplicate `@deprecated` here, maybe combine them somehow?

Reply:
done, I also checked for all duplicates in a commit